src/OrchardCore.Modules/OrchardCore.Localization/Startup.cs

```diff
@@ -50,14 +50,14 @@ public override async ValueTask ConfigureAsync(IApplicationBuilder app, IEndpoin
             var supportedCultures = await localizationService.GetSupportedCulturesAsync();
 
             var cultureOptions = serviceProvider.GetService<IOptions<CultureOptions>>().Value;
+            var localizationOptions = serviceProvider.GetService<IOptions<RequestLocalizationOptions>>().Value;
 
-            app.UseRequestLocalization(options =>
-            {
-                options.CultureInfoUseUserOverride = !cultureOptions.IgnoreSystemSettings;
-                options.SetDefaultCulture(defaultCulture);
-                options.AddSupportedCultures(supportedCultures);
-                options.AddSupportedUICultures(supportedCultures);
-            });
+            localizationOptions.CultureInfoUseUserOverride = !cultureOptions.IgnoreSystemSettings;
+
+            app.UseRequestLocalization(localizationOptions
+                .SetDefaultCulture(defaultCulture)
+                .AddSupportedCultures(supportedCultures)
+                .AddSupportedUICultures(supportedCultures));
```

Comment:
I would first complete the `localizationOptions` configuration outside just before and then call `app.UseRequestLocalization(localizationOptions)` like in the setup startup and I think how it was originally done.

Reply:
I think it's same because it call the chaining methods then return the object

Reply:
IF think it's the same. It's just uglier (in this PR).